mixins/dateTimeUtils.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Utility for converting time between ticks and milliseconds
+ * Utility for converting time between ticks and milliseconds, and formatting dates
```

Comment:
```suggestion
 * Utility for converting time between ticks, milliseconds and formatting dates
```

Reply:
The original one is correct, imo. We're converting between ticks and milliseconds, not between ticks, milliseconds and formatting dates.